docs/guide/controller/installation.md

```diff
@@ -5,8 +5,15 @@
     Please follow our [migration guide](../upgrade/migrate_v1_v2.md) to do migration.
 
 === "Via Helm"
-    Follow the instructions in [aws-load-balancer-controller](https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller) helm chart.
-
+    1. Create a configmap for leadership election:
```

Comment:
The published chart v1.0.0 doesn't need have the `resourceNames` restriction. The next chart release, we will put in a more restricted permissions. So this additional instructions will not be needed.

Reply:
sure, removed :D